docker/test/docker-labels.golden.json

```diff
@@ -11,6 +11,7 @@
   "org.opencontainers.image.description": "Workflow engine for microservice orchestration",
   "org.opencontainers.image.documentation": "https://docs.camunda.io/docs/self-managed/zeebe-deployment/",
   "org.opencontainers.image.licenses": "(Apache-2.0 AND LicenseRef-Zeebe-Community-1.1)",
+  "org.opencontainers.image.ref.name": "ubuntu",
```

Comment:
@npepinpe in this case we inherited a [new label from the base image](https://hub.docker.com/layers/library/eclipse-temurin/17-jre-focal/images/sha256-9702b2b5cdba1aafb1544b5e2de58f6d2b3d6ed4f261b56f5cf479830d74aea4?context=explore) which lead to the verification [to fail](https://github.com/camunda/zeebe/actions/runs/4058577997/jobs/6993445212).

As of now we don't explicitly overwrite this label, thus it only got added to the golden file. I guess it's fine as I don't see a need to overwrite it. Wdyt?

Reply:
Honestly it's not super clear what this label is meant to be used for. Looking at [other docs](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) you can see it mentions that it's often used as a means to differentiate build flavors.

> A common use case of descriptors with a "org.opencontainers.image.ref.name" annotation is representing a "tag" for a container image. For example, an image may have a tag for different versions or builds of the software. In the wild you often see "tags" like "v1.0.0-vendor.0", "2.0.0-debug", etc. Those tags will often be represented in an image-layout repository with matching "org.opencontainers.image.ref.name" annotations like "v1.0.0-vendor.0", "2.0.0-debug", etc.

For us, the version probably fits? Our flavors are then arch based, but that's already clear from the manifest. Having ubuntu there is probably not wrong but doesn't mean much. But as I said, it's quite unclear. AFAIK OpenShift ignores it anyway for its UI, but that might have changed since I last looked.

Reply:
We agreed to use the name of the base image here - "eclipse-temurin:17-jre-focal" - in accordance with the following statement:
`If the image.base.name annotation is specified, the image.base.digest annotation SHOULD be the digest of the manifest referenced by the image.ref.name annotation.`
from https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys